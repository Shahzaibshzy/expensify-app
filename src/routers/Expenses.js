import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";

const Expenses = () => (
  <div>
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default Expenses;
