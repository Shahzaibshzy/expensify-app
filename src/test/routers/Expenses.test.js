import React from "react";
import { shallow } from "enzyme";
import Expenses from "../../routers/Expenses";


test("should test Expenses DAshboard", ()=>{
    const wrapper = shallow(<Expenses/>);
    expect(wrapper).toMatchSnapshot();
});
