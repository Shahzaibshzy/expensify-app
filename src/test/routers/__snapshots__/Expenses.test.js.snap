// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Expenses DAshboard 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
